Octokit/Models/Response/PullRequest.cs

```diff
@@ -136,6 +137,11 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
         public User Assignee { get; protected set; }
 
         /// <summary>
+        ///The multiple users this pull request is assigned to.
+        /// </summary>
+        public IReadOnlyList<User> Assignees { get; protected set; }
```

Comment:
Nic pickup thanks! 

The ctor isn't actually use by the deserializer to create the response objects (it uses the field setters) but from a consistency point on view I should have include t new field in the ctor